.github/workflows/chart.yaml

```diff
@@ -0,0 +1,26 @@
+name: Package and Push Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+      - ci
+
+jobs:
+  release-helm-chart:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Push Helm chart to GHCR
+        uses: bsord/helm-push@v4
```

Comment:
Can you use `helm push` instead of depending on a third-party action ?

Reply:
Sure thing, the action is now using the Helm v4 cli tool.